modules/platform/dev.galasa.platform/build.gradle

```diff
@@ -0,0 +1,132 @@
+
+        api 'dev.galasa:com.auth0.jwt:4.4.1'
+        api 'dev.galasa:dev.galasa.wrapping.io.grpc.java:0.38.0'
+
+        api 'io.kubernetes:client-java:16.0.2'
+
+        // Most bundles before implementing the Platform used 0.6.0 other than dev.galasa.framework.k8s.controller.
+        // Upgrading all to 0.15.0 by using the Platform.
+        api 'io.prometheus:simpleclient:0.15.0'
+        api 'io.prometheus:simpleclient_httpserver:0.15.0'
+        api 'io.prometheus:simpleclient_hotspot:0.15.0'
+
+        api 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.41'
+
+        api 'javax.servlet:javax.servlet-api:3.1.0'
+
+        api 'javax.validation:validation-api:2.0.1.Final'
+
+        api 'junit:junit:4.13.1'
+
+        api 'org.apache.bcel:bcel:6.7.0'
+
+        api 'org.codehaus.plexus:plexus-utils:3.0.24'
+
+        api 'org.apache.commons:commons-collections4:4.4'
+        api 'org.apache.commons:commons-compress:1.26.0'
+        api 'org.apache.commons:commons-lang3:3.14.0'
+
+        api 'org.apache.felix:org.apache.felix.bundlerepository:2.0.2' // If updating, also update in galasa-boot build.gradle.
+        api 'org.apache.felix:org.apache.felix.framework:7.0.5' // If updating, also update in galasa-boot build.gradle.
+        api 'org.apache.felix:org.apache.felix.scr:2.1.14' // If updating, also update in galasa-boot build.gradle.
+
+        api 'org.apache.logging.log4j:log4j-api:2.24.1' // If updating, also update in galasa-boot build.gradle.
+        api 'org.apache.logging.log4j:log4j-core:2.24.1' // If updating, also update in galasa-boot build.gradle.
+        api 'org.apache.logging.log4j:log4j-slf4j-impl:2.24.1'
+
+        api 'org.apache.maven:maven-repository-metadata:3.3.9'
+
+        api 'org.apache.servicemix.bundles:org.apache.servicemix.bundles.okio:2.8.0_1'
+
+        api 'org.apache.tomcat:annotations-api:6.0.53'
+
+        api 'org.assertj:assertj-core:3.16.1'
+        // api 'org.assertj:assertj-core:3.23.1'
+        // 3.23.1 found in dev.galasa.framework.api.cps but breaks Extensions unit tests when used in other bundles.
+        // So using 3.16.1 as the default and 3.23.1 as override in dev.galasa.framework.api.cps.
+
+        api 'org.awaitility:awaitility:3.0.0'
+
+        api 'org.bouncycastle:bcpkix-jdk18on:1.79'
+        api 'org.bouncycastle:bcprov-jdk18on:1.79'
+
+        api 'org.jetbrains.kotlin:kotlin-osgi-bundle:1.4.0'
+
+        api 'org.mockito:mockito-core:3.1.0'
+
+        api 'org.osgi:org.osgi.core:6.0.0'
+        api 'org.osgi:org.osgi.service.cm:1.6.0'
+        api 'org.osgi:org.osgi.service.component.annotations:1.3.0'
+
+        api 'org.yaml:snakeyaml:2.0'
+
+    }
+}
+
+
+publishing {
+    publications {
+        myPlatform(MavenPublication) {
+            from components.javaPlatform
+
+            groupId = 'dev.galasa'
+            artifactId = 'dev.galasa.platform'
+            version = '0.38.0'
+        }
+
+        // mavenBom(MavenPublication) {
```

Comment:
Need a task on the story to publish the maven bom as an artifact, as we need that for the OBR project I think.